### PR TITLE
Fix up some none working parts, update links, and remove out of date nfo

### PR DESCRIPTION
**Pull request by theapplicationist**
More to come

### CODE REVIEW

**Comment by theapplicationist** [comment]
@super3 Here it is

File: header.php [hunk 1]
```diff
@@ -41,13 +41,13 @@
...
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+//     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+//     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+//     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+//     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-    ga('create', 'UA-40768937-6', 'primecoin.io');
-    ga('send', 'pageview');
+//     ga('create', 'UA-40768937-6', 'primecoin.io');
+//     ga('send', 'pageview');
```
Comment (super3) [inline header.php]: Why comment out google analytics?

Comment (theapplicationist) [reply header.php]: I was hosting it elsewhere, but I've re-enabled that for this PR now

**Comment by super3** [comment]
@theapplicationist Read for me to merge this?

**Comment by theapplicationist** [comment]
Yep go ahead! thanks

**Comment by theapplicationist** [comment]
Thanks for doing this!

**Pull request merged**